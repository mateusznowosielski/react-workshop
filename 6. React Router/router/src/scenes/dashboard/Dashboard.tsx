import * as React from 'react';

const Dashboard: React.FunctionComponent<any> = () => {
  return (
    <div>
      <h1>Dashboard</h1>
    </div>
  )
};

export default Dashboard;