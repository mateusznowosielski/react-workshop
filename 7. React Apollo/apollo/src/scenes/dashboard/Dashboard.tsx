import * as React from 'react';
// import { useApolloClient } from 'react-apollo-hooks';
// import { DELETE_BROADCAST } from '../admin/broadcast/queries/queries';

const Dashboard: React.FunctionComponent<any> = () => {
  //
  // const client = useApolloClient();
  //
  // console.log('yay');
  //
  // let message: string = '';
  //
  // // @ts-ignore
  // client.mutate({
  //   mutation: DELETE_BROADCAST,
  //   variables: {
  //     id: 21
  //   },
  // }).then((data: any) => {
  //   console.log('mutate', data);
  //   if (data.error) {
  //     message = 'Error';
  //   }
  // });

  return (
    <div>
      <h1>Dashboard</h1>
    </div>
  )
};

export default Dashboard;