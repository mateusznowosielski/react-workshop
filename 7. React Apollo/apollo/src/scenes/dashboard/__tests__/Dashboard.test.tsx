import * as React from 'react';
import { cleanup, render } from 'react-testing-library';
import Dashboard from 'scenes/dashboard/Dashboard';

afterEach(cleanup);

it('verifying Dashboard to be rendered', () => {
  const { container } = render(<Dashboard/>);

  expect(container).toMatchSnapshot();
});