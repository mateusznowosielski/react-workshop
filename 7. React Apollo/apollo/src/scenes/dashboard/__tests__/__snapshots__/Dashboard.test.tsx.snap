// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`verifying Dashboard to be rendered 1`] = `
<div>
  <div>
    <h1>
      Dashboard
    </h1>
  </div>
</div>
`;
